.github/workflows/ci.yaml

```diff
@@ -130,8 +130,63 @@ jobs:
         git diff --exit-code docs
       continue-on-error: true
 
+  unit-macos:
+    env:
+      TKG_CUSTOM_IMAGE_REPOSITORY: projects-stg.registry.vmware.com/tkg
+      TANZU_CLI_NO_INIT: true
+      TANZU_HOME: $HOME/tanzu
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version: 1.19.x
+    - name: Grype scan
+      run: |
+        echo "##[group]Install grype"
+          curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | sh -s -- -b /usr/local/bin
+        echo "##[endgroup]"
+        echo "##[group]Scan source ."
+          grype dir:. --exclude ./hack,./vendor --exclude '**/testdata/**'
+
+    - name: Install tanzu cli
+      run: |
+        TANZU_VERSION=$(cat TANZU_VERSION)
+        env
```

Comment:
**Question**: Why do we need to list out `env`?

Reply:
This just to see the env variables being set in the CI output. It is inspired from the existing the unit test case for Linux